src/main/resources/templates/db_scripts/schema.json

```diff
@@ -4,11 +4,16 @@
       "run": "after",
       "snippet": "CREATE OR REPLACE VIEW receiving_history_view AS SELECT pieces.id AS id, json_build_object('id', pieces.jsonb->>'id', 'caption', pieces.jsonb->>'caption', 'comment', pieces.jsonb->>'comment', 'dateOrdered', purchase_order.jsonb->>'dateOrdered', 'checkin', po_line.jsonb->>'checkinItems', 'instanceId', po_line.jsonb->>'instanceId','itemId', pieces.jsonb->>'itemId', 'locationId', pieces.jsonb->>'locationId', 'pieceFormat', pieces.jsonb->>'format', 'poLineId', pieces.jsonb->>'poLineId', 'poLineNumber', po_line.jsonb->>'poLineNumber', 'purchaseOrderId', po_line.jsonb->>'purchaseOrderId', 'poLineReceiptStatus', po_line.jsonb->>'receiptStatus','receivedDate', pieces.jsonb->>'receivedDate', 'title', po_line.jsonb->>'title','receivingNote', po_line.jsonb->'details'->>'receivingNote', 'receivingStatus', pieces.jsonb->>'receivingStatus', 'supplement', pieces.jsonb->>'supplement') AS jsonb FROM pieces LEFT OUTER JOIN po_line ON pieces.jsonb->>'poLineId' = po_line.jsonb->>'id' LEFT OUTER JOIN purchase_order ON po_line.jsonb->>'purchaseOrderId' = purchase_order.jsonb->>'id';",
       "fromModuleVersion": "1.0"
-    }
-  ,{
+    },
+    {
       "run": "after",
       "snippet": "CREATE OR REPLACE VIEW orders_view AS SELECT purchase_order.id AS id, purchase_order.jsonb AS jsonb, po_line.jsonb || COALESCE(purchase_order.jsonb, '{}'::jsonb) AS metadata FROM po_line LEFT JOIN purchase_order ON (po_line.jsonb -> 'purchaseOrderId'::text) = (purchase_order.jsonb -> 'id'::text);",
       "fromModuleVersion": "1.0"
+    },
+    {
+      "run": "after",
+      "snippet": "CREATE OR REPLACE VIEW order_lines_view AS SELECT po_line.id AS id, po_line.jsonb AS jsonb, po_line.jsonb || COALESCE(purchase_order.jsonb, '{}'::jsonb) AS metadata FROM po_line LEFT JOIN purchase_order ON (po_line.jsonb -> 'purchaseOrderId'::text) = (purchase_order.jsonb -> 'id'::text);",
+      "fromModuleVersion": "5.0"
```

Comment:
When reviewing #102 I was going to ask Slava to change `fromModuleVersion` for `orders_view` from `1.0` to `5.0` as well. However when I checked version in latest testing vagrant box I was a bit confuced and decided to leave it as it is. However I still think that proper version should be really `5.0` for any new changes...
![image](https://user-images.githubusercontent.com/43410167/56746043-1ceaba80-6784-11e9-9392-02a0b8589ebe.png)

Reply:
That is true but I think RMB decides if it is an upgrade based on the input provided to tenant API, atleast from this line 
https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/src/main/java/org/folio/rest/impl/TenantAPI.java#L244

So even I prefer the proper version to be maintained in the files so that the views are not created atleast in case of older versions.